src/query/service/src/interpreters/access/privilege_access.rs

```diff
@@ -574,6 +574,31 @@ impl AccessChecker for PrivilegeAccess {
         }
         let need_change = user.auth_info.get_need_change();
         if need_change {
+            if let Plan::Query {
+                metadata,
+                rewrite_kind: None,
+                ..
+            } = plan
+            {
+                // Some simple SQL used by BendSQL is allowed,
+                // such as: `select version()`, `select * from system.tables`
+                let mut allowed = true;
+                let metadata = metadata.read().clone();
+                for table in metadata.tables() {
+                    let db_name = table.database();
+                    let table_name = table.name();
+                    if !((db_name == "system"
+                        && SYSTEM_TABLES_ALLOW_LIST.iter().any(|x| x == &table_name))
+                        || db_name == "information_schema")
+                    {
+                        allowed = false;
```

Comment:
cc @wubx Please review this. MySQL user can do simple select?

Reply:
```sql
SELECT version()

select name from system.tables union all select name from system.columns union all select name from system.databases union all select name from system.functions
```

When using `BendSQL` client to login, those two SQL will be automatically executed. If the privilege check fails here, `BendSQL` will be panic, and user can't change his password.

Reply:
have delete this logic, we modify `BendSQL` to fix the problem